source/connectors/kafka-sink-properties.txt

```diff
@@ -0,0 +1,249 @@
+.. _kafka-sink-properties:
+
+=============================================
+Kafka Sink Connector Configuration Properties
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Sink Connector Configuration Properties
+---------------------------------------
+
+Refer to the table below to create or migrate a properties file for your
+MongoDB Kafka sink connector. See `MongoSinkConnector.properties
+<https://github.com/mongodb/mongo-kafka/blob/master/config/MongoSinkConnector.properties>`_
+for an example configuration file.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 1 1 3
+
+   * - Name
+     - Type
+     - Description
+
+   * - topics
+     - list
+     - | A list of Kafka topics for the sink connector.
+       |
+       | **Default**: (none)
+       | **Accepted Values**: A non-empty list
+
+   * - connection.uri
+     - string
+     - | A :manual:`standard MongoDB connection URI string </reference/connection-string/#standard-connection-string-format>`.
+       | For example: ``mongodb://username:password@localhost/``
+       |
+       | **Default**: ``mongodb://localhost:27017``
+       | **Accepted Values**: A valid connection string
```

Comment:
I could not find any documentation that indicated it would be optional -- will need to do some research into determining this. Will check out the Query Filters page and create an example snippet.